sdk/monitor/azure-monitor-opentelemetry-autoconfigure/README.md

```diff
@@ -0,0 +1,199 @@
+# Azure Monitor OpenTelemetry Exporter client library for Java
```

Comment:
it's ok to do as a follow up before we release, but we should mark exporter as deprecated and redirect people to autoconfigure